### PR TITLE
BehaviorSpace Sub Experiment Bug Fixes

### PR DESCRIPTION
**Pull request by ERSUCC**
Removed accidental debug statement and fixed issue with constant redefine checking

### CODE REVIEW

**Comment by brandesNW** [comment]
The previous examples meet expectations. However the following should produce a constant redefined error.

```
["a" [1 3 5]]
["a" 33]
[["b" [6 1 9]]]
```

gives

```
[run number]	a	b	[step]
1	1	6	1
2	1	7	1
3	1	8	1
4	1	9	1
5	4	6	1
6	4	7	1
7	4	8	1
8	4	9	1
```

the line `["a" 33] `seems to just be ignored.

**Comment by brandesNW** [comment]
There are places in the files
netlogo-gui/resources/i18n/GUI_Strings_en.properties
netlogo-gui/src/main/lab/gui/ProtocolEditable.scala
that need to be reconciled with hexy
If you get all of them the conflicts may go away.

File: netlogo-gui/resources/i18n/GUI_Strings_en.properties [hunk 1]
```diff
@@ -335,7 +335,7 @@ edit.plot.error.runtimeError = Runtime Error:
 edit.plot.copyimage = Copy Image
 edit.plot.clearplot = Clear
 
-edit.behaviorSpace.list.increment = Increment list for "{0}" exceeds the number of possible runs {1}
+edit.behaviorSpace.list.increment = Increment list "{0}" exceeds the number of possible runs
```
Comment (brandesNW) [inline netlogo-gui/resources/i18n/GUI_Strings_en.properties:338]: take value from hexy

File: netlogo-gui/src/main/lab/gui/ProtocolEditable.scala [hunk 2]
```diff
@@ -150,12 +150,12 @@ class ProtocolEditable(protocol: LabProtocol,
...
               if (more.isEmpty) {complain(I18N.gui.getN("edit.behaviorSpace.expectedValue", variableName)); return None}
               val exp = new RefEnumeratedValueSet(variableName, more.toList)
-              if (subExperiment.contains(exp)) {
+              if (subExperiment.exists(_.variableName == variableName)) {
                 complain(I18N.gui.getN("edit.behaviorSpace.variableDefinedTwiceSubexperiment", variableName)); return None
               }
               subExperiment = subExperiment :+ exp
             case _ =>
-              complain(I18N.gui.get("edit.behaviorSpace.invalidFormat") + (List(first.toList) ++ more)); return None
+              complain(I18N.gui.get("edit.behaviorSpace.invalidFormat")); return None
```
Comment (brandesNW) [inline netlogo-gui/src/main/lab/gui/ProtocolEditable.scala:158]: take value from hexy

**Review by brandesNW — CHANGES_REQUESTED** [review]

**Comment by brandesNW** [comment]
my testing was successful

**Comment by brandesNW** [comment]
The netlogo-gui/resources/i18n/GUI_Strings_en.properties change is good.
Isaac and Ryan don't get a conflict when they view it (because of their repo?)
Maybe it can't be rebased because of the order in which things were integrated.

**Review by brandesNW — APPROVED** [review]

**Comment by ERSUCC** [comment]
Issue with error messages resolved. Let me know if it is better now.

**Review by LaCuneta — APPROVED** [review]
LGTM :+1:

**Pull request merged**